packages/langium/src/references/scope-computation.ts

```diff
@@ -73,8 +73,9 @@ export class DefaultScopeComputation implements ScopeComputation {
         this.descriptions = services.workspace.AstNodeDescriptionProvider;
     }
 
-    async computeExports(document: LangiumDocument, cancelToken = CancellationToken.None): Promise<AstNodeDescription[]> {
-        return this.computeExportsForNode(document.parseResult.value, document, undefined, cancelToken);
+    async computeExports(document: LangiumDocument, cancelToken?: CancellationToken): Promise<AstNodeDescription[]> {
+        const token = cancelToken ?? CancellationToken.None;
```

Comment:
What's the issue with the default parameter value syntax?

Reply:
No issue per se - I think I just had interface alignment in mind after the previous exercise (#1604 ).

Reply:
See my comment https://github.com/eclipse-langium/langium/pull/1604#discussion_r1696977473 – I prefer the previous parameter initialization syntax over this modified variant.

Reply:
Reverted the change. 👍